website/static/sh/install

```diff
@@ -137,12 +142,15 @@ install_benthos()
 	esac
 	chmod +x "$PREFIX/tmp/$benthos_bin"
 
-	echo "Putting benthos in $install_path (may require password)"
-	$sudo_cmd mv "$PREFIX/tmp/$benthos_bin" "$install_path/$benthos_bin"
+	echo "Putting benthos in $benthos_install_path (may require password)"
+	$sudo_cmd mv "$PREFIX/tmp/$benthos_bin" "$benthos_install_path/$benthos_bin"
 	$sudo_cmd rm -- "$dl"
 
 	# check installation
-	$benthos_bin -version
+	$benthos_install_path/$benthos_bin -version
```

Comment:
When installing with `asdf` benthos is not on the path yet (this happens when you run `asdf global ...`.

Means we need to distinguish between we can execute the bin and it's presence on the path.